wsgi/odyssy/odyssy/settings.py

```diff
@@ -114,3 +114,17 @@
 # https://docs.djangoproject.com/en/1.8/howto/static-files/
 STATIC_URL = '/static/'
 STATIC_ROOT = os.path.join(WSGI_DIR, 'static')
+
+
+# This will allow cookies  to be sent only via HTTPS connections.
+SESSION_COOKIE_SECURE = True
+CSRF_COOKIE_SECURE = True
+
+
+# Terminates the session as soon as browser closes.
+SESSION_EXPIRE_AT_BROWSER_CLOSE = True
```

Comment:
I agree with the fact that most of users will have personal devices, but when we are designing a security model, we can't proceed by assuming average or best case scenario. We have to assume worst case scenario. I have taken example of a cyber cafe but there are more general cases like logging in on a friend's laptop. I suggest we should go with maximum viable security.

@singhpratyush Setting expiry time is also a nice idea. We could implement that.